Disable the 0x0188 user_ext_flags entry. The vendor build does not write it
(CFG_USER_EXT_FLAGS_EN is 0).

--- a/src/SDIO/driver_fw/driver/aic8800/aic8800_bsp/aic8800d80_compat.c
+++ b/src/SDIO/driver_fw/driver/aic8800/aic8800_bsp/aic8800d80_compat.c
@@ -82,6 +82,7 @@
 	{0x00000170, 0x0000010a}, //irqf
 #endif
 
+#if 0 /* vendor does not write 0x0188 */
     {0x0188,
         (USER_EXT_FLAGS_DEFAULT_D80 |
             #if CFG_USER_CHAN_MAX_TXPWR_EN
@@ -114,6 +115,7 @@
             #endif
         0)
     }, // user_ext_flags
+#endif
 
 #ifdef CONFIG_RADAR_OR_IR_DETECT
 	{0x019c, 0x00000900}, //enable radar detect
